plugins/modules/get_certificate.py

```diff
@@ -99,6 +99,14 @@
         - The default value V(false) is B(deprecated) and will change to V(true) in community.crypto 3.0.0.
       type: bool
       version_added: 2.12.0
+    ssl_ctx_options:
```

Comment:
Please use TLS instead of SSL. The protocol has been named TLS for 25 years by now.
```suggestion
    tls_ctx_options:
```